js/app/restaurant/components/Order/MobileOrderHeading.js

```diff
@@ -39,7 +42,7 @@ export default function MobileOrderHeading() {
         }) }
         onClick={ handleClick }>
         <div className="panel-heading__body">
-          { isMobileCartVisible ? null : (<OrderState />) }
+          { isMobileCartVisible ? (<span>{t('CART_TITLE')}</span>) : (<OrderState />) }
```

Comment:
it is what I see when I have the mobile cart closed, is it expected? (display warning about order total) 

![image](https://github.com/coopcycle/coopcycle-web/assets/4426585/ea8dcce8-dd20-46df-94f6-cf600029c680)

Reply:
Yes, it comes from the `<OrderState />` component